sesearch

```diff
@@ -144,6 +144,9 @@ try:
     p = setools.SELinuxPolicy(args.policy)
 
     if args.tertypes:
+        self_rule = args.target == "self"
+        if self_rule:
+            args.target = None
```

Comment:
I'd rather have an explicit `--self` option that conflicts with the `-t` option (see the above checks that call parser.error() for example). It could probably conflict with the regex and indirect options for the target too, though I'm not sure that needs to be checked.